### ISSUE TITLE
【EESS】New submissions for Mon,  4 Mar 24

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#1223]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Phase retrieval beyond the homogeneous object assumption for X-ray  in-line holographic imaging
 - **Authors:** Jens Lucht, Leon M. Lohse, Thorsten Hohage, Tim Salditt
 - **Subjects:** Image and Video Processing (eess.IV); Optics (physics.optics)
 - **Arxiv link:** https://arxiv.org/abs/2403.00461
 - **Pdf link:** https://arxiv.org/pdf/2403.00461
 - **Abstract**
 X-ray near field holography has proven to be a powerful 2D and 3D imaging technique with applications ranging from biomedical research to material sciences. To reconstruct meaningful and quantitative images from the measurement intensities, however, it relies on computational phase retrieval which in many cases assumes the phase-shift and attenuation coefficient of the sample to be proportional. Here, we demonstrate an efficient phase retrieval algorithm that does not rely on this homogeneous-object assumption and is a generalization of the well-established contrast-transfer-function (CTF) approach. We then investigate its stability and present an experimental study comparing the proposed algorithm with established methods. The algorithm shows superior reconstruction quality compared to the established CTF-based method at similar computational cost. Our analysis provides a deeper fundamental understanding of the homogeneous object assumption and the proposed algorithm will help improve the image quality for near-field holography in biomedical applications
### Structure Similarity Preservation Learning for Asymmetric Image  Retrieval
 - **Authors:** Hui Wu, Min Wang, Wengang Zhou, Houqiang Li
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2403.00648
 - **Pdf link:** https://arxiv.org/pdf/2403.00648
 - **Abstract**
 Asymmetric image retrieval is a task that seeks to balance retrieval accuracy and efficiency by leveraging lightweight and large models for the query and gallery sides, respectively. The key to asymmetric image retrieval is realizing feature compatibility between different models. Despite the great progress, most existing approaches either rely on classifiers inherited from gallery models or simply impose constraints at the instance level, ignoring the structure of embedding space. In this work, we propose a simple yet effective structure similarity preserving method to achieve feature compatibility between query and gallery models. Specifically, we first train a product quantizer offline with the image features embedded by the gallery model. The centroid vectors in the quantizer serve as anchor points in the embedding space of the gallery model to characterize its structure. During the training of the query model, anchor points are shared by the query and gallery models. The relationships between image features and centroid vectors are considered as structure similarities and constrained to be consistent. Moreover, our approach makes no assumption about the existence of any labeled training data and thus can be extended to an unlimited amount of data. Comprehensive experiments on large-scale landmark retrieval demonstrate the effectiveness of our approach. Our code is released at: https://github.com/MCC-WH/SSP.
### Asymmetric Feature Fusion for Image Retrieval
 - **Authors:** Hui Wu, Min Wang, Wengang Zhou, Zhenbo Lu, Houqiang Li
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2403.00671
 - **Pdf link:** https://arxiv.org/pdf/2403.00671
 - **Abstract**
 In asymmetric retrieval systems, models with different capacities are deployed on platforms with different computational and storage resources. Despite the great progress, existing approaches still suffer from a dilemma between retrieval efficiency and asymmetric accuracy due to the limited capacity of the lightweight query model. In this work, we propose an Asymmetric Feature Fusion (AFF) paradigm, which advances existing asymmetric retrieval systems by considering the complementarity among different features just at the gallery side. Specifically, it first embeds each gallery image into various features, e.g., local features and global features. Then, a dynamic mixer is introduced to aggregate these features into compact embedding for efficient search. On the query side, only a single lightweight model is deployed for feature extraction. The query model and dynamic mixer are jointly trained by sharing a momentum-updated classifier. Notably, the proposed paradigm boosts the accuracy of asymmetric retrieval without introducing any extra overhead to the query side. Exhaustive experiments on various landmark retrieval datasets demonstrate the superiority of our paradigm.
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Go Beyond Black-box Policies: Rethinking the Design of Learning Agent  for Interpretable and Verifiable HVAC Control
 - **Authors:** Zhiyu An, Xianzhong Ding, Wan Du
 - **Subjects:** Systems and Control (eess.SY); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2403.00172
 - **Pdf link:** https://arxiv.org/pdf/2403.00172
 - **Abstract**
 Recent research has shown the potential of Model-based Reinforcement Learning (MBRL) to enhance energy efficiency of Heating, Ventilation, and Air Conditioning (HVAC) systems. However, existing methods rely on black-box thermal dynamics models and stochastic optimizers, lacking reliability guarantees and posing risks to occupant health. In this work, we overcome the reliability bottleneck by redesigning HVAC controllers using decision trees extracted from existing thermal dynamics models and historical data. Our decision tree-based policies are deterministic, verifiable, interpretable, and more energy-efficient than current MBRL methods. First, we introduce a novel verification criterion for RL agents in HVAC control based on domain knowledge. Second, we develop a policy extraction procedure that produces a verifiable decision tree policy. We found that the high dimensionality of the thermal dynamics model input hinders the efficiency of policy extraction. To tackle the dimensionality challenge, we leverage importance sampling conditioned on historical data distributions, significantly improving policy extraction efficiency. Lastly, we present an offline verification algorithm that guarantees the reliability of a control policy. Extensive experiments show that our method saves 68.4% more energy and increases human comfort gain by 14.8% compared to the state-of-the-art method, in addition to an 1127x reduction in computation overhead. Our code and data are available at https://github.com/ryeii/Veri_HVAC
### Spatio-temporal reconstruction of substance dynamics using compressed  sensing in multi-spectral magnetic resonance spectroscopic imaging
 - **Authors:** Utako Yamamoto, Hirohiko Imai, Kei Sano, Masayuki Ohzeki, Tetsuya Matsuda, Toshiyuki Tanaka
 - **Subjects:** Signal Processing (eess.SP); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2403.00402
 - **Pdf link:** https://arxiv.org/pdf/2403.00402
 - **Abstract**
 The objective of our study is to observe dynamics of multiple substances in vivo with high temporal resolution from multi-spectral magnetic resonance spectroscopic imaging (MRSI) data. The multi-spectral MRSI can effectively separate spectral peaks of multiple substances and is useful to measure spatial distributions of substances. However it is difficult to measure time-varying substance distributions directly by ordinary full sampling because the measurement requires a significantly long time. In this study, we propose a novel method to reconstruct the spatio-temporal distributions of substances from randomly undersampled multi-spectral MRSI data on the basis of compressed sensing (CS) and the partially separable function model with base spectra of substances. In our method, we have employed spatio-temporal sparsity and temporal smoothness of the substance distributions as prior knowledge to perform CS. The effectiveness of our method has been evaluated using phantom data sets of glass tubes filled with glucose or lactate solution in increasing amounts over time and animal data sets of a tumor-bearing mouse to observe the metabolic dynamics involved in the Warburg effect in vivo. The reconstructed results are consistent with the expected behaviors, showing that our method can reconstruct the spatio-temporal distribution of substances with a temporal resolution of four seconds which is extremely short time scale compared with that of full sampling. Since this method utilizes only prior knowledge naturally assumed for the spatio-temporal distributions of substances and is independent of the number of the spectral and spatial dimensions or the acquisition sequence of MRSI, it is expected to contribute to revealing the underlying substance dynamics in MRSI data already acquired or to be acquired in the future.
### Phase retrieval beyond the homogeneous object assumption for X-ray  in-line holographic imaging
 - **Authors:** Jens Lucht, Leon M. Lohse, Thorsten Hohage, Tim Salditt
 - **Subjects:** Image and Video Processing (eess.IV); Optics (physics.optics)
 - **Arxiv link:** https://arxiv.org/abs/2403.00461
 - **Pdf link:** https://arxiv.org/pdf/2403.00461
 - **Abstract**
 X-ray near field holography has proven to be a powerful 2D and 3D imaging technique with applications ranging from biomedical research to material sciences. To reconstruct meaningful and quantitative images from the measurement intensities, however, it relies on computational phase retrieval which in many cases assumes the phase-shift and attenuation coefficient of the sample to be proportional. Here, we demonstrate an efficient phase retrieval algorithm that does not rely on this homogeneous-object assumption and is a generalization of the well-established contrast-transfer-function (CTF) approach. We then investigate its stability and present an experimental study comparing the proposed algorithm with established methods. The algorithm shows superior reconstruction quality compared to the established CTF-based method at similar computational cost. Our analysis provides a deeper fundamental understanding of the homogeneous object assumption and the proposed algorithm will help improve the image quality for near-field holography in biomedical applications
### Optimization of the Energy-Comfort Trade-Off of HVAC Systems in Electric  City Buses Based on a Steady-State Model
 - **Authors:** Fabio Widmer, Stijn van Dooren, Christopher H. Onder
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2403.00517
 - **Pdf link:** https://arxiv.org/pdf/2403.00517
 - **Abstract**
 The electrification of public transport vehicles offers the potential to relieve city centers of pollutant and noise emissions. Furthermore, electric buses have lower life-cycle greenhouse gas (GHG) emissions than diesel buses, particularly when operated with sustainably produced electricity. However, the heating, ventilation, and air-conditioning (HVAC) system can consume a significant amount of energy, thus limiting the achievable driving range. In this paper, we address the HVAC system in an electric city bus by analyzing the trade-off between the energy consumption and the thermal comfort of the passengers. We do this by developing a dynamic thermal model for the bus cabin, which we simplify by considering it to be in steady state. We introduce a method that is able to quickly optimize the steady-state HVAC system inputs for a large number of samples representative of a year-round operation. A comparison between the results from the steady-state optimization approach and a dynamic simulation reveal small deviations in both the HVAC system power demand and achieved thermal comfort. Thus, the approximation of the system performance with a steady-state model is justified. We present two case studies to demonstrate the practical relevance of the approach. First, we show how the method can be used to compare different system designs based on a year-round performance evaluation. Second, we show how the method can be used to generate accurate setpoints for online controllers. In conclusion, this study shows that a steady-state analysis of the HVAC systems of an electric city bus is a valuable approach to evaluate and optimize its performance.
### User Localization with HRIS and Backscatter Modulation for  Next-Generation Networks
 - **Authors:** Mattia Piana, Stefano Tomasin
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2403.00568
 - **Pdf link:** https://arxiv.org/pdf/2403.00568
 - **Abstract**
 Hybrid reflective intelligent surfaces (HRISs) can support localization in sixth-generation (6G) networks thanks to their ability to generate narrow beams and at the same time receive and process locally the impinging signals. In this paper, we propose a novel protocol for user localization in a network with an HRIS. The protocol includes two steps. In the first step, the HRIS operates in full absorption mode and the user equipment (UE) transmits a signal that is locally processed at the HRIS to estimate the angle of arrival (AoA). In the second step, the base station transmits a downlink reference signal to the UE, and the HRIS superimposes a message by a backscatter modulation. The message contains information on the previously estimated AoA. Lastly, the UE, knowing the position of the HRIS, estimates the time of flight (ToF) from the signal of the second step and demodulates the information on the AoA to obtain an estimate of its location. Numerical results confirm the effectiveness of the proposed solution, also in comparison with the Cram\'er Rao lower bound on the estimated quantities.nd on the estimated quantities.
### Graph Theory and GNNs to Unravel the Topographical Organization of Brain  Lesions in Variants of Alzheimer's Disease Progression
 - **Authors:** Leopold Hebert-Stevens, Gabriel Jimenez, Benoit Delatour, Lev Stimmer, Daniel Racoceanu
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2403.00636
 - **Pdf link:** https://arxiv.org/pdf/2403.00636
 - **Abstract**
 This study utilizes graph theory and deep learning to assess variations in Alzheimer's disease (AD) neuropathologies, focusing on classic (cAD) and rapid (rpAD) progression forms. It analyses the distribution of amyloid plaques and tau tangles in postmortem brain tissues. Histopathological images are converted into tau-pathology-based graphs, and derived metrics are used for statistical analysis and in machine learning classifiers. These classifiers incorporate SHAP value explainability to differentiate between cAD and rpAD. Graph neural networks (GNNs) demonstrate greater efficiency than traditional CNN methods in analyzing this data, preserving spatial pathology context. Additionally, GNNs provide significant insights through explainable AI techniques. The analysis shows denser networks in rpAD and a distinctive impact on brain cortical layers: rpAD predominantly affects middle layers, whereas cAD influences both superficial and deep layers of the same cortical regions. These results suggest a unique neuropathological network organization for each AD variant.
### Degradation-Invariant Music Indexing
 - **Authors:** Rémi Mignot, Geoffroy Peeters (LTCI)
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2403.00688
 - **Pdf link:** https://arxiv.org/pdf/2403.00688
 - **Abstract**
 For music indexing robust to sound degradations and scalable for big music catalogs, this scientific report presents an approach based on audio descriptors relevant to the music content and invariant to sound transformations (noise addition, distortion, lossy coding, pitch/time transformations, or filtering e.g.). To achieve this task, one of the key point of the proposed method is the definition of high-dimensional audio prints, which are intrinsically (by design) robust to some sound degradations. The high dimensionality of this first representation is then used to learn a linear projection to a sub-space significantly smaller, which reduces again the sensibility to sound degradations using a series of discriminant analyses. Finally, anchoring the analysis times on local maxima of a selected onset function, an approximative hashing is done to provide a better tolerance to bit corruptions, and in the same time to make easier the scaling of the method.
